release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1171,6 +1171,10 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 
 * Previously, the procedural name generator for Azure availability sets exceeded the 80 character maximum limit. This might cause the Machine API to reuse the same sets during the name truncation, rather than creating multiple availability sets. With this release, the procedural name generator is updated to ensure that the name is no longer than 80 characters and the cluster name is not duplicated in the set name. As a result, Azure availability sets are no longer truncated in unexpected ways by the procedural name generator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2093044[*BZ#2093044*])
 
+* Because the Cluster Autoscaler Operator was deploying the cluster autoscaler without setting any leader election parameters, the cluster autoscaler could unexpectedly fail and restart after a cluster restart. With this fix, the Cluster Autoscaler Operator now deploys the cluster autoscaler with well-defined leader election flags. As a result, the cluster autoscaler operates as expected after restarts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2063194[*BZ#2063194*])"
```

Comment:
When referring to the (Cluster Autoscaler) Operator, initial caps are used and when not talking about the Operator, it is all lowercase...just want to confirm?

Reply:
I just did a search, looks like maybe lowercase?? so cluster autoscaler Operator. Going to wait for confirmation.

Reply:
"cluster autoscaler" and "Cluster Autoscaler Operator"

- https://docs.openshift.com/container-platform/4.10/machine_management/applying-autoscaling.html
- https://docs.openshift.com/container-platform/4.10/operators/operator-reference.html#cluster-autoscaler-operator_platform-operators-ref